Contributing.md update

NidhiKumari2510 opened the pull request:
improved CONTRIBUTING.md formatting and structure

AditiGupta-tech commented:
Hey @NidhiKumari2510 , great job! Your PR has been reviewed and it has a few missing sections as expected from the tasks provided in the issue description [View here](https://github.com/AditiGupta-tech/neonest/issues/7).

1. PR review process
2. File naming conventions (for folders, images, etc.)
3. Links (GitHub Discussions link, discord, etc.)
4. Link to `CODE_OF_CONDUCT.md` (you can make a placeholder for now).

Kindly complete these changes and get your first PR merged!

NidhiKumari2510 commented:
@AditiGupta-tech thank you for you feedback 
I have already added PR review process and file naming convention.
<img width="539" height="534" alt="image" src="https://github.com/user-attachments/assets/c2e55515-d2e3-451d-afb1-4f869e72310b" />

I have added GitHub discussion link a s well but I am not sure about which link should I add for discord.
<img width="916" height="409" alt="image" src="https://github.com/user-attachments/assets/9e9c3e26-b3f0-42b1-b5a3-abe06a144665" />

I have also added a small section code_of_conduct as well.
please check it and let me know if you want me to change anything.

AditiGupta-tech commented:
> @AditiGupta-tech thank you for you feedback I have already added PR review process and file naming convention. <img alt="image" width="539" height="534" src="https://private-user-images.githubusercontent.com/192850455/471909181-c2e55515-d2e3-451d-afb1-4f869e72310b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM3ODEzMzUsIm5iZiI6MTc1Mzc4MTAzNSwicGF0aCI6Ii8xOTI4NTA0NTUvNDcxOTA5MTgxLWMyZTU1NTE1LWQyZTMtNDUxZC1hZmIxLTRmODY5ZTcyMzEwYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyOVQwOTIzNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNjNlYTVlMGFkZWFkZjE3MGYxZDczZDVjNGI0YzIzMWE1YTVmMThkNGMzYTUxYzRiMWJiYTFkMjkxOWRkNWIxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.0OFs4-2x50t1OQGVLElzTYLFx3q5Bwu_MpMrHbDXx_A">
> 
> I have added GitHub discussion link a s well but I am not sure about which link should I add for discord. <img alt="image" width="916" height="409" src="https://private-user-images.githubusercontent.com/192850455/471910454-9e9c3e26-b3f0-42b1-b5a3-abe06a144665.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM3ODEzMzUsIm5iZiI6MTc1Mzc4MTAzNSwicGF0aCI6Ii8xOTI4NTA0NTUvNDcxOTEwNDU0LTllOWMzZTI2LWIzZjAtNDJiMS1iNWEzLWFiZTA2YTE0NDY2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyOVQwOTIzNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZmI1YzEzMjViMmUwOGNhNDI5ZGFmOWE1NDI4MzY5MzRjN2M0MjU2YzBjMGM5MjcwN2Q5OWIxMmQxYjBjOWZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.lWM_OwhkLxEBw-ZQtkzPY3X-8LPMDp4i0Kgy_c_oshQ">
> 
> I have also added a small section code_of_conduct as well. please check it and let me know if you want me to change anything.

No worries, u can add the placeholder links for now.

also, I am unable to view changes in the file as per your commits. Make sure to double-check if you've pushed them to the correct branch, then either update your current PR or create a new one
